chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+if [ "$#" -eq 0 ]; then
+  echo "Usage: chapter2-t2-gen-kustomization.sh [REPO]..."
+  exit 1
+fi
+
+readonly DEPLOY_KEY_POSTFIX="deploy-key.pem"
+
+# stop on first error
+set -e
+
+# init, prepare kustomize
+mkdir -p repos
+echo \
```

Comment:
It is more common to use `cat` for writing more then one line to a file instead of `echo` - https://linuxhint.com/what-is-cat-eof-bash-script/